pandas/tests/indexing/test_indexing.py

```diff
@@ -72,7 +55,7 @@ def test_setitem_ndarray_1d(self):
         with pytest.raises(ValueError, match=msg):
             df[2:5] = np.arange(1, 4) * 1j
 
-    @pytest.mark.parametrize("idxr", [getitem, loc, iloc])
+    @pytest.mark.parametrize("idxr", [tm.getitem, tm.loc, tm.iloc])
```

Comment:
worth making indexer_gli (though maybe a more meaningful name)

Reply:
maybe down the road.  i want to see where else we can use these patterns, and if we can get e.g. at/iat at the same time